lib/dgram.js

```diff
@@ -140,6 +147,12 @@ function startListening(socket) {
   socket._receiving = true;
   socket._bindState = BIND_STATE_BOUND;
   socket.fd = -42; // compatibility hack
+  
+  if (socket._recvBufferSize)
```

Comment:
Ah, right, sorry. I was reading the diff as if the changes were in the same function. Might be a good case for #5496 though.